lib/command/base.rb

```diff
@@ -372,10 +394,34 @@ def cp
       @cp ||= Controlplane.new(config)
     end
 
-    def perform(cmd)
+    def perform!(cmd)
```

Comment:
Just for consistency. We generally add `!` at the end of the method name if it raises an error.